src/Microsoft.Azure.NotificationHubs/TemplateNotification.cs

```diff
@@ -15,6 +15,7 @@ namespace Microsoft.Azure.NotificationHubs
     /// </summary>
     public sealed class TemplateNotification : Notification
     {
+        const string contentType = "application/json";
```

Comment:
Templates should be `application/json;charset=utf-8` according to [our own documentation](https://docs.microsoft.com/en-us/rest/api/notificationhubs/send-template-notification)